src/Stripe.net/Services/TaxRates/TaxRateCreateOptions.cs

```diff
@@ -0,0 +1,54 @@
+namespace Stripe
+{
+    using System;
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+    using Stripe.Infrastructure;
+
+    public class TaxRateCreateOptions : BaseOptions
+    {
+        /// <summary>
+        /// Whether the tax rate is currently available for new subscriptions.
+        /// </summary>
+        [JsonProperty("active")]
+        public bool? Active { get; set; }
+
+        /// <summary>
+        /// An arbitrary string attached to the object. Often useful for displaying to users.
```

Comment:
I think `description`'s description got mixed up with `display_name`'s.

This should be: "An arbitrary string attached to the tax rate for your internal use only. It will not be visible to your customers."